src/.github/workflows/wheels.yml

```diff
@@ -0,0 +1,23 @@
+name: Build Python Wheels
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+  jobs:
```

Comment:
This should be indented one level less

Reply:
:scream: yaml is the worst

Reply:
given you're building python only makes sense indentation plays a role :P screw `{}`, right?